### ISSUE TITLE
Let us contribute on cobalt 9 

### ISSUE THREAD

**Issue by andiamine** [Serhioromano/cobalt-9#40]
Hi i can't fork cobalt 9 it's empty, i want really contribute by adding uikit views, and other things

**Comment by pepperstreet** [comment]
IMHO, it is not enough to think about templates/views.
The most crucial part for display issues and compatibility are **fields** and **forms**!
In most cases fields might use _JS_, more complex markup and special formatting.
This has a big impact and needs a lot of thinking and (re-)work, right?

**Comment by kiranetph** [comment]
@andiamine this is a public repository without the source code. Its only purpose is to use the free issue tracker, therefore no "fork" of code. 
But you have your local copy based on the releases here, so whatever you want to contribute, you can post it under a separate issue for public discussion or sent it directly to Sergey for his evaluation.
I think he will highly appreciate any help/contribution.

**Comment by andiamine** [comment]
@kiranetph it's hard for me to work like that, adding source code here will help me to make changes quickly and pull it, then @Serhioromano can review it.

**Comment by Serhioromano** [comment]
@andiamine I agree that contribute through GitHub PR is very convenient. But I cannot open source may development branch.

If you send it to me, I have a tool to compare and evaluate changes. If I find something and change it, you will get that with next update.

I think right now the only way for you is to create files in your current installation. Those should not be overridden on updates. Start with this and then we will see how to simplify merge process.

**Comment by andiamine** [comment]
yes i agree with dev branch, also if you don't want share all your code or you want a private repository let's try then https://bitbucket.org/product/features

**Comment by Serhioromano** [comment]
I have private repository with cobalt here on GitHub. Let's start with something you create. Start creating and then we will see how we can organize process.

**Comment by andiamine** [comment]
good so i will start adding uikit in cobalt9 , i will add view prefixed first

can i access this private repository ?
